### ISSUE TITLE
com.jcabi.dynamo.Throughput 

### ISSUE THREAD

**Issue by yegor256** [jcabi/jcabi-dynamo#10]
migrated from https://github.com/jcabi/jcabi/issues/276

Let's implement a class `com.jcabi.dynamo.Throughput`, which would be able to change throughput of a table according to certain parameters of elasticity/scalability. For example:

```java
final Region region = //...
final Table table = region.table("my-users");
new Throughput(table).adjust();
```

The class is intended to be executed regularly (every few minutes). It has to use the same AWS credentials to access CloudWatch metrics.

**Comment by dmarkov** [comment]
@yegor256 right, I added the `bug` tag

**Comment by dmarkov** [comment]
we will find someone to do this task, soon

**Comment by dmarkov** [comment]
@yegor256 many thanks, **15 mins** added to your acc for reporting this bug, pmt ID `000-ec5c2eb1`

**Comment by dmarkov** [comment]
@tousifkhan please pick this up, and keep in mind [these instructions](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). Any technical questions - ask right here. The budget here is **30 mins**, which is exactly how much time will be paid for, when the task is [completed](http://www.xdsd.org/2014/04/15/definition-of-done.html)

**Comment by dmarkov** [comment]
@tousifkhan you're working with this ticket for the last 15 days. If it is not closed within the next 48 hours, it will be re-assigned to someone else, see [No Obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle. [This article](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html) should help if you're stuck... -30 added to your rating, current score is: [-30](http://www.netbout.com/b/35305?open=rating)

**Comment by dmarkov** [comment]
@tousifkhan it takes too long. Usually, we expect any task to be finished in less than a week. I'll assign someone else. Please stop working with it right now. See our [no obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle; added -60 to your rating, now it is equal to [-120](http://www.netbout.com/b/35305?open=rating)

**Comment by dmarkov** [comment]
@suseika could you please pick this up? This [article](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) explains how we work. Any technical questions you may ask right here

This task's budget is **30 mins**. This is exactly how much will be paid when the problem explained above is solved. See [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) for more information

**Comment by Suseika** [comment]
@yegor256 
> would be able to change throughput of a table
> has to use the same AWS credentials to access CloudWatch metrics

Aren't these two distinct responsibilities that should be assigned to distinct classes?

**Comment by Suseika** [comment]
@yegor256 Is this implementation ok?

**Comment by yegor256** [comment]
@Suseika yeah, looks like a good starting point, thanks!

**Comment by Suseika** [comment]
@dmarkov The task is completed.

**Comment by Suseika** [comment]
@yegor256 Is this ready to be closed?

**Comment by yegor256** [comment]
@Suseika in which pull request was it fixed?

**Comment by Suseika** [comment]
@yegor256 Ah, sorry, you're right, it wasn't. #45 is just a starting point.

**Comment by yegor256** [comment]
@Suseika thanks!

**Comment by Suseika** [comment]
@dmarkov Please pay for this task.

**Comment by Suseika** [comment]
@dmarkov This problem is solved with #45

**Comment by dmarkov** [comment]
@suseika I just added **38 mins** to your account, many thanks for your contribution (AP-6FF45764YP578332R).. 70 hours and 13 mins spent here. the bonus for fast delivery (m=4213). +38 to your rating, your total score is [+128](http://www.netbout.com/b/35381?open=rating)

**Comment by Suseika** [comment]
@dmarkov Thanks!

**Comment by dmarkov** [comment]
@yegor256 the code made here contains a puzzle `10-d74a245f`/#46, which will be resolved soon

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#unknown]() is still not solved.

**Comment by 0pdd** [comment]
@yegor256 all 2 puzzles are solved here: [#88](https://github.com/jcabi/jcabi-dynamo/issues/88), [#unknown]().
